packages/nimble-blazor/NimbleBlazor/Components/NimbleTextArea.razor.cs

```diff
@@ -29,6 +29,12 @@ public partial class NimbleTextArea : NimbleInputBase<string?>
     [Parameter]
     public string? Placeholder { get; set; }
 
+    [Parameter]
+    public bool? ErrorVisible { get; set; }
+
+    [Parameter]
+    public string? ErrorText { get; set; }
```

Comment:
Let's add the standard tests for these properties as well.

Reply:
Added